chapter_preface/index_vn.md

```diff
@@ -42,7 +52,17 @@ changing the way movies are made, diseases are diagnosed,
 and playing a growing role in basic sciences---from astrophysics to biology.
 -->
 
-*dịch đoạn phía trên*
+Trong chỉ khoảng năm năm gần đây, học sâu đã mang đến nhiều bất ngờ trên quy mô toàn cầu, 
+dẫn đường cho những tiến triển nhanh chóng trong nhiều lĩnh vực khác
+nhau như thị giác máy tính, xử lý ngôn ngữ tự nhiên, nhận dạng giọng nói tự động,
+học tăng cường, và mô hình hoá thống kê. Với những tiến bộ này, chúng ta bây
```

Comment:
Tương tự ý trên: 

thị giác máy tính (Tiếng Anh: computer vision), xử lý ngôn ngữ tự nhiên (Tiếng Anh: natural language processing, NLP), nhận dạng giọng nói tự động (Tiếng Anh: automatic speech recognition), học tăng cường (Tiếng Anh: reinforcement learning), và mô hình hoá thống kê (Tiếng Anh: statistical modeling)

Reply:
Mình thấy không cần thêm "Tiếng Anh" làm gì cả vì ai cũng biết nó là tiếng Anh rồi.

Reply:
Phần tiếng Anh cũng để in nghiêng để hiểu đó là lời giải thích.